frontend/src/components/elements/PlotlyChart/PlotlyChart.test.tsx

```diff
@@ -77,18 +92,42 @@ describe("PlotlyChart Element", () => {
       }
       const wrapper = mount(<PlotlyChart {...props} />)
 
-      expect(wrapper.find(Plot).props()).toMatchSnapshot()
+      expect(wrapper.find(Plot).props().layout.width).toBe(400)
+      expect(wrapper.find(Plot).props().layout.height).toBe(400)
     })
 
-    it("useContainerWidth", () => {
+    it("useContainerWidth true", () => {
       const props = {
         ...getProps({
           useContainerWidth: true,
         }),
-        width: 400,
       }
       const wrapper = mount(<PlotlyChart {...props} />)
-      expect(wrapper.find(Plot).props()).toMatchSnapshot()
+
+      // an explicit value because useContainerWidth is passed
+      expect(wrapper.find(Plot).props().layout.width).not.toBeUndefined()
+      expect(wrapper.find(Plot).props().layout.height).toBeUndefined()
+    })
+
+    it("useContainerWidth false", () => {
+      const props = {
+        ...getProps({
+          useContainerWidth: false,
+        }),
+      }
+      const wrapper = mount(<PlotlyChart {...props} />)
+
+      // an explicit value because useContainerWidth is passed
```

Comment:
I will remove this comment because it's wrong but don't want to interrupt snapshots.